main/src/library/Concurrent/Channel.flix

```diff
@@ -63,14 +63,14 @@ namespace Concurrent/Channel {
     /// structure on which select can operate.
     ///
     @Internal
-    pub enum Mpmc[a](
+    pub enum Mpmc[a: Type, _: Region](
```

Comment:
I think you have two options here:

- Either you just drop the region entirely (and ignore it) or
- You introduce the region, but then you should also use it, i.e. in the definition of MutDeque.

Reply:
If you keep the region parameter then you should use the actual region available in Lowering.

Reply:
The right thing would be to use the region, but the easy thing (which I lean towards), is to ignore it.